src/User/Service/MailService.php

```diff
@@ -83,11 +83,17 @@ public function getType()
      */
     public function run()
     {
-        return $this->mailer
+
+        $result =  $this->mailer
```

Comment:
`$result =   $this->mailer`
`        ^^^^ whitespace`

Reply:
I have made the change here